### DISCUSSION TITLE
Some docstring headers (numpy/scipy/etc) are too big in tooltips

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/pylance-release#2694] [Enhancement]
This is the hover that I see: 

![image](https://user-images.githubusercontent.com/2473545/95807876-990ef000-0ce1-11eb-91f5-51a570a0a1ff.png)

It has giant fonts. 
I'd rather not have a hover than to have this.

**Comment by ghost** [comment]
And of course pylance is closed source, that means the community can't fix the problem.

**Comment by Bluppsn** [comment]
I think that's intended, numpy has formatted docstrings too.

![image](https://user-images.githubusercontent.com/67192819/95853055-0d36ab80-0d55-11eb-9fa0-3a6a01299308.png)

**Comment by ghost** [comment]
@Bluppsn:

Here's what I get with Jedi:
![image](https://user-images.githubusercontent.com/2473545/95858267-35151780-0d33-11eb-85f1-2dd788331828.png)


With Jedi, it looks much more sane.

**Comment by jakebailey** [comment]
See also #48.

We do a loose translation of docstrings; these is restructured text headers, which we do (mostly) handle, but the first header size in markdown (what VS Code renders) is an HTML h1, which is of course large. This could be improved.

(This code is not closed, and is a straight port from MPLS: https://github.com/microsoft/pyright/blob/master/packages/pyright-internal/src/analyzer/docStringConversion.ts)

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
